SwrveSDK/src/main/java/com/swrve/sdk/SwrveBase.java

```diff
@@ -805,7 +807,7 @@ private boolean checkCampaignRules(int elementCount, String elementName, String
     @SuppressLint("UseSparseArrays")
     protected SwrveConversation _getConversationForEvent(String event) {
         SwrveConversation result = null;
-        SwrveCampaign campaign = null;
+        SwrveMessageCampaign campaign = null;
```

Comment:
Is it strange that we use the SwrveMessageCampaign in this method for conversations? Am confused. (Sorry, I know this isn't directly part of your code changes)